### PR TITLE
test(JestMigration): convert Area

### PR DESCRIPTION
**Pull request by Yilun-Sun**
## Description

Refactor `test/specs/cartesian/AreaSpec.js` -> `test-jest/cartesian/Area.spec.ts`

## Related Issue

#3158 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: test-jest/cartesian/Area.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+      return <circle key={cx} x={cx} y={cy} r={5} className="customized-dot" />;
+    };
+    const { container } = render(
+      <Surface width={500} height={500}>
+        <Area dataKey="value" isAnimationActive={false} baseLine={200} points={data} dot={renderDot} />
+      </Surface>,
+    );
+
+    expect(container.querySelectorAll('.customized-dot')).toHaveLength(data.length);
```
Comment (nikolasrieble) [inline test-jest/cartesian/Area.spec.tsx:70]: Nit: You could consider to follow best practice guide and query via `screen` as opposed to container. 

https://stackoverflow.com/questions/61482418/react-testing-library-screen-vs-render-queries

File: test-jest/cartesian/Area.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,120 @@
+import React, { FC } from 'react';
+import { render } from '@testing-library/react';
+import { Area, Surface } from '../../src';
+
+describe('<Area />', () => {
+  const data: any = [
```
Comment (nikolasrieble) [inline test-jest/cartesian/Area.spec.tsx:6]: It would be nice to not use any, just to set an example.

**Review by nikolasrieble — APPROVED** [review]
Looks good! I left two minor nits you could consider.

**Review by nikolasrieble — APPROVED** [review]
🚀

**Review by ckifer — CHANGES_REQUESTED** [review]
LGTM 🚀 .  One thing @Yilun-Sun could you squash this into 2 commits? One commit specifically for the change to Area props and one for the tests. Then I'll rebase merge so that the commit history is preserved nicely for the two different changes

**Comment by ckifer** [comment]
@Yilun-Sun thanks! Merged the other PR but looks like there's still 5 commits here with conflicts. You'll have to reset to master, pull down the area props change, then commit the test changes in 1 commit rather than 5

**Review by ckifer — APPROVED** [review]

**Pull request merged**